gradle/libs.versions.toml

```diff
@@ -16,6 +16,9 @@ recyclerview = "1.3.2"
 viewpager2 = "1.1.0"
 target_sdk_version = "34"
 fragment = "1.8.4"
+junitVersion = "1.2.1"
+espressoCore = "3.6.1"
+material = "1.12.0"
```

Comment:
I can't find usages of those new dependencies, could you remove them if they're not used please? :)